### PR TITLE
docs: VirtualizedList and VirtualizedGrid related components should use Text instead of inline string

### LINKED ISSUE #2504: Change `VirtualizedList` catalog component to use `Text` component
In the file of:
`packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx`
which can be seen on our [Catalog](https://vibe.monday.com/?path=/docs/catalog--docs) page, we do not currently use Vibe's `Text` component, but instead use inline string.
It also causes theme issues where a story doesn't look right on dark themes.

We need to change the wrapper `div` to be `Text` instead.
```typescript
<div
  className={styles.virtualizedListItem}
  style={{
    backgroundColor,
    height: item.height
  }}
>
  {item.value}
</div>
```
--> Something like
```typescript
<Text
  className={styles.virtualizedListItem}
  color={Text.colors.FIXED_DARK}
  style={{
    backgroundColor,
    height: item.height
  }}
>
  {item.value}
</Text>
```

Notice the `FIXED_DARK` color addition as well - we need it so it would be shown correct on all themes.

monday ticket:
https://monday.monday.com/boards/3532714909/pulses/7080412658

### PR DESCRIPTION
**Pull request by GurneeshBudhiraja**
Changes made:
Replaced the div inside the VirtualizedList catalog component with the Text component.

- [X] I have read the [Contribution Guide](../packages/core/CONTRIBUTING.md) for this project.

Resolves #2504 #2505

### CODE REVIEW

File: packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx [hunk 1]
```diff
@@ -3,22 +3,24 @@ import { RelatedComponent } from "vibe-storybook-components";
 import VirtualizedList from "../../../../../components/VirtualizedList/VirtualizedList";
 import { generateItems } from "../../../../../components/VirtualizedList/__stories__/VirtualizedList.stories.helpers";
 import styles from "./virtualized-list.module.scss";
+import { Text } from "src/components";
```
Comment (YossiSaadi) [inline packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx:6]: same as in the previous PR, the import here is wrong
please run Storybook to make sure your changes work

**Review by YossiSaadi — CHANGES_REQUESTED** [review]
Thanks @GurneeshBudhiraja , can you please fix here as well?

**Comment by YossiSaadi** [comment]
@GurneeshBudhiraja let me know if you're planning on fixing this one as well :)

**Comment by GurneeshBudhiraja** [comment]
> @GurneeshBudhiraja let me know if you're planning on fixing this one as well :)

Almost done.

**Review by YossiSaadi — APPROVED** [review]
💯 !

**Comment by YossiSaadi** [comment]
you accidentally inserted the two commits of Grid and List in the same PR
I'm closing the #2534 and resolving the two issues on this PR
marked the other PR as hacktoberfest-accepted to make sure you'll get points for both PRs

**Pull request merged**